### DISCUSSION TITLE
Delta printer z_offset problem in Klipper screen

### DISCUSSION THREAD

**Discussion by Mike-Tomcat** [KlipperScreen/KlipperScreen#987] [Q&A]
Hello people. Im getting a very strange behavior in a new Delta Style 3D printer. I' using Klipper Screen from the rasp and when i set the probe_z_offset in the screen, its saves correctly but if i call the tool head to positon 0 it smashes agaist the bed! If i do the same procedure in the console of the PC this problem is gone!. Is there any kind of offset loaded by klipper screen that im not seeing?

Many thanks

**Comment by alfrix** [comment]
> is there any kind of offset loaded by klipper screen that im not seeing

no klipperscreen doesn't apply any offset, i'm not familiar with deltas, but the issue you seem to be experiencing seems to be related to the fact that klipper doesn't reset the gcode z_offset after a calibration (also called babystepping by other firmware) this is not an issue with klipperscreen, as it is an issue with klipper itself, usually not a problem since most users save_config after a probe_calibrate which resets said offset. also i have been told that a delta_calibrate is necessary after an offset change, again i can't confirm this since i don't have a delta
